dev/Dockerfile

```diff
@@ -2,10 +2,9 @@ FROM php:7.4-cli
 
 USER root
 
-RUN apt update && apt install libgearman-dev zip unzip gearman-tools bc procps jq -y && pecl install gearman
+RUN apt update && apt install zip unzip bc procps jq apache2-utils -y
```

Comment:
Why do we need to add `apache2-utils`?

Reply:
It's needed for the Apache benchmark used in `ci-performance` bash script. With `ab` added to docker we can run CI tests inside the container.